tests/gis_tests/layermap/models.py

```diff
@@ -15,6 +15,11 @@ class State(NamedModel):
     pass
 
 
+class Province(NamedModel):
+    bag_id = models.CharField(max_length=16, primary_key=True)
+    border = models.MultiPolygonField(null=True)
```

Comment:
Can you add `null=True` to `County` or do you need a new model? We try to avoid adding new models if possible since it slows the tests.

Reply:
If you're ok with not having a MultiPolygon field without null available (and thus not being able to test on required enforcement), then it's fine with me and I will refactor to use that model.

Reply:
I don't see any test failures with that change so it seems fine.